SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/ConnectionConfig.h

```diff
@@ -122,6 +117,35 @@ class FLocatorConfig : public FConnectionConfig
 	FString LoginToken;
 };
 
+class FDevAuthConfig : public FConnectionConfig
+{
+public:
+	FDevAuthConfig()
+	{
+		LoadDefaults();
+	}
+
+	void LoadDefaults()
+	{
+		UseExternalIp = true;
+		LocatorHost = SpatialConstants::LOCATOR_HOST;
+	}
+
+	bool TryLoadCommandLineArgs()
+	{
+		bool bSuccess = true;
+		const TCHAR* CommandLine = FCommandLine::Get();
+		FParse::Value(CommandLine, TEXT("locatorHost"), LocatorHost);
+		FParse::Value(CommandLine, TEXT("deployment"), Deployment);
+		bSuccess = FParse::Value(CommandLine, TEXT("devAuthToken"), DevelopmentAuthToken);
+		return bSuccess;
+	}
+
+	FString LocatorHost;
```

Comment:
That's not the first case this happens, but is it intentional, that `FString LocatorHost` is also duplicated in base class `FConnectionConfig`?

Reply:
I just checked the base class, but I don't think we have the locator host in there. It is currently duplicated for both the LocatorConfig and the DevAuthConfig as they both need to connect via the locator. I have been thinking about whether it would make sense to merge those two or have them inherit from the same class, but they only share the LocatorHost, so have kept them separate for now.
Could change that though, I don't have a strong opinion on this.

Reply:
Ah, my bad, looked at a wrong line :( I don't think there's a need to change it for now, resolving.